improper version expression in peer dependency

Hi.

Recent version of `package.json` has critical problem to describe the version ranges of `@nestjs/graphql` package.

When I tried to add recent version of this package with my project using @nestjs/graphql@10.0.0,

```bash
➜  *** git:(release/v0.2) ✗ npm install nest-access-control --save
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: nest-access-control@2.1.0
npm WARN Found: @nestjs/graphql@10.0.6
npm WARN node_modules/@nestjs/graphql
npm WARN   peer @nestjs/graphql@"^10.0.0" from @nestjs/apollo@10.0.6
npm WARN   node_modules/@nestjs/apollo
npm WARN     @nestjs/apollo@"^10.0.0" from the root project
npm WARN   1 more (the root project)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer @nestjs/graphql@"^10.0.0" from @nestjs/apollo@10.0.6
npm WARN node_modules/@nestjs/apollo
npm WARN   @nestjs/apollo@"^10.0.0" from the root project
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: nest-access-control@2.1.0
npm WARN Found: @nestjs/graphql@10.0.6
npm WARN node_modules/@nestjs/graphql
npm WARN   peer @nestjs/graphql@"^10.0.0" from @nestjs/apollo@10.0.6
npm WARN   node_modules/@nestjs/apollo
npm WARN     @nestjs/apollo@"^10.0.0" from the root project
npm WARN   1 more (the root project)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peerOptional @nestjs/graphql@"^7.0.0 | ^8.0.0 | ^9.0.0 | ^10.0.0" from nest-access-control@2.1.0
npm WARN node_modules/nest-access-control
npm WARN   nest-access-control@"*" from the root project
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: nest-access-control@2.1.0
npm WARN Found: @nestjs/graphql@undefined
npm WARN node_modules/@nestjs/graphql
npm WARN   @nestjs/graphql@"^10.0.0" from the root project
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peerOptional @nestjs/graphql@"^7.0.0 | ^8.0.0 | ^9.0.0 | ^10.0.0" from nest-access-control@2.1.0
npm WARN node_modules/nest-access-control
npm WARN   nest-access-control@"*" from the root project
```

According to syntax definition of dependencies section in `package.json` from official document (https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies), 

the version of dependent package which allow multiple versions should divided by `||` operator, not `|`.

Currently, `packages.json` describe the version ranges of `@nestjs/graphql` using `|` operator.

```json
  "peerDependencies": {
    "@nestjs/graphql": "^7.0.0 | ^8.0.0 | ^9.0.0 | ^10.0.0"
  },
  "peerDependenciesMeta": {
    "@nestjs/graphql": {
      "optional": true
    }
  }
```

--- a/package.json
+++ b/package.json
@@ -51,7 +51,7 @@
     "accesscontrol": "^2.2.1"
   },
   "peerDependencies": {
-    "@nestjs/graphql": "^7.0.0 | ^8.0.0 | ^9.0.0 | ^10.0.0"
+    "@nestjs/graphql": "^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0"
   },
   "peerDependenciesMeta": {
     "@nestjs/graphql": {

